### DISCUSSION TITLE
Can VASP can be used to do fp calculation in dpgen init_reaction?

### DISCUSSION THREAD

**Discussion by ZSHtju** [deepmodeling/dpgen#1141] [Q&A]
Can VASP can be used to do fp calculation in dpgen init_reaction? 
If VASP can be used, then what are the differences between VASP and Gaussian in these types of fp calculations?

**Comment by AnguseZhang** [comment]
I think currently `init_reaction` does not support for VASP style. 

I think that Initially it was designed for preparing data for reaction (combustion etc) , in which case we more focus small molecules and Gaussian is more commonly used than VASP. 

If you still want use VASP,  some modifications're needed and PRs are welcome.
